### ISSUE TITLE
Missing xml sax, how to integrate ?

### ISSUE THREAD

**Issue by Xavier74** [nvdaaddons/DevGuide#1]
hello,

I'm trying to build an addon to manage sonos devices via soco api. when importing soco library, soco/core.py try to load xml.sax and cannot find it. it seems to load the xml found into library.zip in nvda subfolder, and does not look in the xml folder i've included in my addon and customized with the sys.path.insert(0, 'path\\to\\xml')
have you a solution to override the xml included into the library.zip package? or to customise it if it's better.

many thanks in advance for your help.

best regards

**Comment by josephsl** [comment]
Hi, although not really a question for this repo… One possible workaround would be including Python XML package inside your add-on and rename the package folder and importing things from there. Thanks.

**Comment by Xavier74** [comment]
thanks for your reply, i don't have solved the problem. I don't know how to do. if you have a repos where i can ask... ?

**Comment by CyrilleB79** [comment]
@josephsl wrote:
> Hi, although not really a question for this repo… (...)

Actually it seems to me that it is a good question for this repo. Such question has been asked many times on NVDA add-on mailing list. Some people wanted to use in there add-on a standard Python module that does not belong to NVDA's library.zip selection. And at the end, it seemed to me that those people did not succeed in integrating correctly these modules.
If it is doable, it would really be interesting to document it in the dev guide. If not yet doable, it should be documented too an issue should be raised against NVDA (if not already existing).

**Comment by josephsl** [comment]
Hi, perhaps it could be part of a future dev guide wiki edits? Let’s talk about this on add-ons list. Thanks.
